examples/regression/1-quickstart/plot_prefit.py

```diff
@@ -0,0 +1,201 @@
+"""
+===========================================================================
+Example use of the prefit parameter with neural networks and LGBM Regressor
+===========================================================================
+
+:class:`mapie.regression.MapieRegressor` and
+:class:`mapie.quantile_regression.MapieQuantileRegressor`
+are used to calibrate uncertainties for large models for
+which the cost of cross-validation is too high. Typically,
+neural networks rely on a single validation set.
+
+In this example, we first fit a neural network on the training set. We
+then compute residuals on a validation set with the `cv="prefit"` parameter.
+Finally, we evaluate the model with prediction intervals on a testing set.
+We will also show how to use the prefit method in the comformalized quantile
+regressor.
+"""
+
+
+import numpy as np
+from lightgbm import LGBMRegressor
+from matplotlib import pyplot as plt
+import scipy
+from sklearn.model_selection import train_test_split
+from sklearn.neural_network import MLPRegressor
+
+from mapie.regression import MapieRegressor
+from mapie.quantile_regression import MapieQuantileRegressor
+from mapie.metrics import regression_coverage_score
+from mapie._typing import NDArray
+import warnings
```

Comment:
Move this import before the mapie importations ?

Reply:
done